spacy/pipeline/pipes.pyx

```diff
@@ -3,278 +3,47 @@
 # coding: utf8
 from __future__ import unicode_literals
 
-import numpy
```

Comment:
If you didn't rename this file, the change would be backward compatible 😇

Reply:
How so? It's not like we documented `import numpy from spacy.pipeline` was supposed to work :p

Reply:
I think he meant the whole file in general? But in terms of user-facing API, it should be backwards compatible. Unless I forgot something here?

https://github.com/explosion/spaCy/blob/196f5e8a2bd68c8cfc2b882e2491a62621e9f5ca/spacy/pipeline/__init__.py#L4-L8

Reply:
Oh my bad. It would just break if you imported components directly from the file, which maybe you care less about. I'll butt out!

Reply:
No worries – definitely appreciate the attention to detail, though!